docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>Maybe add a comment after line 3 stating that <code>x</code> is unchanged. (And then you'd need to add another last line so that the Colab output contains the changed array), e.g.: (maybe there's a better way to do this)</p><p> </p><pre class="ql-syntax" spellcheck="false">x = jnp.array([[1.0, 2.0, 3.0, 4.0],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;[5.0, 6.0, 7.0, 8.0]])
updated = jax.ops.index_update(x,(0,0), 3.0) # whereas x[0,0] = 3.0 would fail

print("x", x)
print("updated", updated)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Added !</p>